### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by glennphilp** [bqworks/slider-pro#184]
Is there any plans to enhance Slider Pro with keyboard input? 

As part of my organization, we must adhere to Section 508 and W3C accessibility guidelines. And the only glitch to using Slider Pro is it doesn't have any keyboard inputs for switching between slides and accessing the content once inside a slide.

**Comment by davidghi** [comment]
Hi. It might not be all that you need, but you can actually navigate between slides by using the left and right arrow keys, and also navigate to links by hitting the Enter key.

Cheers!

**Comment by Adysone** [comment]
Hello,

It would be great if it was possible to disable only the "hitting Enter key to follow the link" part.

I'd like to keep the arrow keys navigation, but right now, I can't submit a form which is on the same page (like a search engine) with the "Enter Key".

Should I open a new issue?

Thank you for your work btw.

**Comment by davidghi** [comment]
Hi,

You can set the `keyboardOnlyOnFocus` option to `true`, so that the slider will be navigable by keyboard input only if the slider has focus. Or you can disable this module, by setting `keyboard` to `false` and then add this code:

```javascript
$( document ).on( 'keydown', function( event ) {
	if ( event.which === 37 ) {
		$('.slider-pro').sliderPro('previousSlide');
	} else if ( event.which === 39 ) {
		$('.slider-pro').sliderPro('nextSlide');
	}
});
```

You're welcome!

**Comment by Adysone** [comment]
Thank you for the tip!
